### PR TITLE
(Feature) finalized savings UI

### PR DESCRIPTION
**Pull request by roma-hladilka**
# Description

merge savings from different networks, add additional column fro network 

About #260 

Screenshot 2022-12-15 at 11 35 02
Screenshot 2022-12-15 at 11 35 27
Screenshot 2022-12-15 at 11 35 44

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/components/Savings/SavingsModal/index.tsx [hunk 1]
```diff
@@ -103,8 +103,8 @@ const SavingsModal = memo(
         useEffect(() => {
```
Comment (johnsmith-gooddollar) [inline src/components/Savings/SavingsModal/index.tsx:103]: I see this effect just does calculation. if setBalance / setWithdrawAmount aren't used anywhere else, could be replaced with useMemo

File: src/pages/gd/Stake/Savings/SavingsDepositMobile.tsx [hunk 2]
```diff
@@ -7,77 +7,80 @@ import { CellSC } from '../styled'
...
+                                <Title type="category" className="flex items-center title">
+                                    {item.title} <QuestionHelper text={item.questionText || ''} />
+                                </Title>
+                            </div>
+                            <div className="font-bold value">
+                                {error ? (
+                                    <LoadingPlaceHolder />
+                                ) : (
+                                    stats &&
+                                    (() => {
```
Comment (johnsmith-gooddollar) [inline src/pages/gd/Stake/Savings/SavingsDepositMobile.tsx:46]: move it to the nested component

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]

**Comment by johnsmith-gooddollar** [comment]
@L03TJ3 awaiting Your review

**Review by L03TJ3 — APPROVED** [review]

**Comment by L03TJ3** [comment]
Only, I know there is an issue on dev servers with loading icons, if just can be verified that the token icons are loaded correctly when on production build

File: src/components/Savings/SavingsModal/index.tsx (hunk 1 above)
Comment (roma-hladilka) [reply src/components/Savings/SavingsModal/index.tsx]: replaced

File: src/pages/gd/Stake/Savings/SavingsDepositMobile.tsx (hunk 2 above)
Comment (roma-hladilka) [reply src/pages/gd/Stake/Savings/SavingsDepositMobile.tsx]: done

File: src/pages/gd/Stake/Savings/index.tsx [hunk 3]
```diff
@@ -16,32 +16,107 @@ import { SavingsDepositMobile } from './SavingsDepositMobile'
...
+        <>
+            <tr>
+                <td>
+                    <AsyncTokenIcon
+                        address={g$.address}
+                        chainId={g$.chainId}
+                        className={'block w-5 h-5 mr-2 rounded-lg md:w-10 md:h-10 lg:w-12 lg:h-12'}
+                    />
+                </td>
+                <td>{i18n._(t`G$`)}</td>
```
Comment (johnsmith-gooddollar) [inline src/pages/gd/Stake/Savings/index.tsx:51]: isn't this similar to SavingsDepositMobile ? let's reuse components from there

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]

File: src/pages/gd/Stake/Savings/index.tsx (hunk 3 above)
Comment (roma-hladilka) [reply src/pages/gd/Stake/Savings/index.tsx:51]: @johnsmith-gooddollar 
here is table layout and in SavingsDepositMobile is flex layout

Comment (roma-hladilka) [reply src/pages/gd/Stake/Savings/index.tsx:51]: or do you mean that SavingsMobileStat component?

Comment (roma-hladilka) [reply src/pages/gd/Stake/Savings/index.tsx]: done

**Review by johnsmith-gooddollar — APPROVED** [review]

**Pull request merged**